integration-cli/docker_cli_build_test.go

```diff
@@ -7357,3 +7357,19 @@ func (s *DockerSuite) TestBuildWindowsEnvCaseInsensitive(c *check.C) {
 		c.Fatalf("Case insensitive environment variables on Windows failed. Got %s", res)
 	}
 }
+
+func (s *DockerSuite) TestBuildLabelsArgs(c *check.C) {
```

Comment:
Probably this can be moved to UT, but I think it is ok to keep this IT at the moment.